src/containers/post/post.component.jsx

```diff
@@ -1,7 +1,14 @@
-import React from "react";
+import React, { useState } from "react";
 import { voteForOption } from "database/votePost";
 
 const Post = (props) => {
+  const [voteView, setVoteView] = useState(false);
+  //console.log(props);
+
+  const handleVote = (obj) => {
```

Comment:
Can have a better variable name than `obj`. Always best to have meaningful variables names for code readability